src/components/TeachersPage/TeachersItem/TeachersItem.tsx

```diff
@@ -8,14 +8,15 @@ import { CustomButton } from '../../../UI-components/CustomButton/CustomButton.t
 import { ButtonSize, ButtonType } from '../../../helpers/types/types.ts'
 import { CustomModal } from '../../../UI-components'
 import { TeacherPopUp } from '../TeacherPopUp/TeacherPopUp.tsx'
-import { FavoriteIcon, NonFavoriteIcon } from '../../../assets/icons'
 import { selectIsUserLoggedIn } from '../../../redux/auth/auth.slice.ts'
 import { useAppSelector } from '../../../helpers/hooks/useAppSelector.ts'
 import {
   addToFavorite,
   removeFromFavorite
 } from '../../../redux/teachers/teachers.slice.ts'
 import { useAppDispatch } from '../../../helpers/hooks/useAppDispatch.ts'
+import toast from 'react-hot-toast'
+import { FavoriteIcon, NonFavoriteIcon } from '../../../assets/icons'
 
 interface TeacherItemProps {
   teacher: Teacher
```

Comment:
```suggestion
interface Props {
  teacher: Teacher
```
By default, you need to write just 'type Props', if its simple component